src/material/paginator/paginator.ts

```diff
@@ -82,6 +82,9 @@ export class MatPaginator extends _MatPaginatorBase implements OnInit, OnDestroy
   HasInitialized {
   private _initialized: boolean;
   private _intlChanges: Subscription;
+  
+  /**to allow overwrite the enable/disable tooltip for previous/next buttons*/
```

Comment:
There should be whitespace in-between the start and end of the comment.

I'm also suggesting that the comment be renamed to this:

```suggestion
  /** Overrides the ability for the tooltip to be disabled for previous/next buttons */
```